GitUI/CommandsDialogs/SettingsDialog/Pages/SshSettingsPage.cs

```diff
@@ -27,15 +26,13 @@ public static SettingsPageReference GetPageReference()
         {
             return new SettingsPageReferenceByType(typeof(SshSettingsPage));
         }
-        private ConfigFileSettings GlobalConfigFileSettings { get { return CommonLogic.ConfigFileSettingsSet.GlobalSettings; } }
 
         protected override void SettingsToPage()
         {
             PlinkPath.Text = AppSettings.Plink;
             PuttygenPath.Text = AppSettings.Puttygen;
             PageantPath.Text = AppSettings.Pageant;
             AutostartPageant.Checked = AppSettings.AutoStartPageant;
-            GitCredPath.Text = GlobalConfigFileSettings.GetValue("credential.helper");
```

Comment:
Maybe we should display warning if credential helper not installed?
Without it HTTPS wouldn't work
I don't know what SuggestGitCred should do after removing git-credential-winstore from installer